package.json

```diff
@@ -51,6 +51,7 @@
     "express": "^4.14.0",
     "express-session": "^1.14.2",
     "extract-text-webpack-plugin": "beta",
+    "fast-json-patch": "^1.1.3",
```

Comment:
# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
# yarn lockfile v1

 how i do this ? I really don't know.